### PR TITLE
Fix guiscrcpy use Python generic icon in Windows taskbar

### PR DESCRIPTION
**Pull request by levirs565**
This wil fix guiscrcpy use Python generic icon in Windows taskbar. This also will fix guiscrcpy use same taskbar with other Python application in Windows.

### CODE REVIEW

**Comment by srevinsaju** [comment]
Wow! You are being an amazing contributor to `guiscrcpy`. Thanks for this second patch, and for testing and fixing bugs in `guiscrcpy` for Windows. :sparkles:

File: guiscrcpy/cli.py [hunk 1]
```diff
@@ -133,6 +133,13 @@ def cli(
         # it will log the traceback to the terminal
         # but it would not be visible to users without CLI interface
         # enable High DPI scaling
+
+        if platform.system() == "Windows":
+            import ctypes
+
+            appid = "srevinsaju.guiscrcpy"
+            ctypes.windll.shell32.SetCurrentProcessExplicitAppUserModelID(appid)
```
Comment (srevinsaju) [inline guiscrcpy/cli.py:141]: It would be nice to have a inline comment explaining that these lines are meant for what, etc.

Comment (levirs565) [reply guiscrcpy/cli.py:141]: You can see this https://stackoverflow.com/a/1552105/9986755

**Comment by srevinsaju** [comment]
Thanks!

**Pull request merged**